science/verilator/Portfile

```
@@ -12,16 +12,18 @@ platforms               darwin
 description             Verilog compiler and simulator
 long_description        Verilator is a {*}${description}.
 
-checksums               rmd160  35d03a6436e3cb7a70a3efc31eb10ee186ab920d \
-                        sha256  6f4322ebe0a863045d5ff08a9bbd90e1c0f658c3d00cd207b7372a498b0002e6 \
-                        size    2816274
+checksums               rmd160  05154dade12df2664a87e774bf49ac7121a8f80d \
+                        sha256  804e3affb9c6f80c97c02ce9fc58a4c79e6fdfcacadd8f8e27374428a6dd42da \
+                        size    3457342
 
 compiler.cxx_standard   2014
 
 use_autoconf            yes
 
 installs_libs           no
 
+depends_lib-append      port:help2man
```

Is `help2man` needed on to build Verilator, or must it be present for Verilator to function?

If it's the former, then this should be `depends_build-append` instead:

```suggestion
depends_build-append    port:help2man
```

---

I believe it's just needed for building, thanks!